src/Web/Views/Account/CustomAssessment.cshtml

```diff
@@ -28,6 +28,8 @@
                                 <div class="card-header">
                                     <div>Use <b>@@deviceFingerprintingId</b> in your request to have it replaced with the DFP Fingerprinting session ID.</div>
                                     <div>Use <b>@@deviceIpAddress</b> in your request to have it replaced with the client's IP address.</div>
+                                    <div>Use <b>@@guid</b> in your request to have it replaced with a randomly generated guid.</div>
+                                    <div>Use <b>@@timestampNow</b> in your request to have it replaced with the current time (ISO format).</div>
```

Comment:
Would 'utcNow' be a more representative name?

Reply:
Yea, I think that would be better. I've renamed it.